client/api/src/backend.rs

```diff
@@ -169,6 +169,15 @@ pub trait BlockImportOperation<Block: BlockT> {
 	fn mark_head(&mut self, id: BlockId<Block>) -> sp_blockchain::Result<()>;
 }
 
+/// Interface for performing operations on the backend.
+pub trait ClientBackend<Block: BlockT, B: Backend<Block>> {
```

Comment:
The name of the trait and its only function don't really match up. May `LockedImportRun` would be a better naming?

Not super happy about that name but it is def cleaner, unless you intent to add other functionality to it.

```suggestion
pub trait LockedImportRun<Block: BlockT, B: Backend<Block>> {
```